### PR TITLE
Add docs for Mongo migration

### PR DESCRIPTION
**Pull request by pingvinen**

### CODE REVIEW

File: pages/mongo/mongo_migration_getting_started.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+
+```csharp
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
+
+namespace rapidcore_migrations
+{
+    public class Quote
+    {
+        [BsonIgnoreIfDefault]
```
Comment (MartinsRui) [inline pages/mongo/mongo_migration_getting_started.md:56]: Should we explain what this is: ``BsonIgnoreIfDefault`?

Comment (pingvinen) [reply pages/mongo/mongo_migration_getting_started.md:56]: I would say no in this case, as this is a getting started guide that should not distract the user with other information than necessary.

File: pages/mongo/mongo_migration_intro.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+The framework currently supports upgrading and has been prepared for downgrading, but we still have not figured out exactly how we want the user experience of downgrading to work. If you have any input, [help us out](https://github.com/rapidcore/issues/issues/14).
+
+## Migrations and MongoDB - are you drunk?
+
+Many will argue that you do not need migrations when you use MongoDB as Mongo itself is schema-less.
+
+As we see it, your data is still following a schema - it is merely enforced by the application rather than the database. Either you will have to support multiple versions of your "schema" in your application or you can write code that transforms the data from your previous schema to the new schema.
+
+A migration should also take of the data itself and not only be about the structure of the data.
```
Comment (MartinsRui) [inline pages/mongo/mongo_migration_intro.md]: *should also take care

Comment (MartinsRui) [reply pages/mongo/mongo_migration_intro.md]: @pingvinen just this one thing here, the rest is fine

**Pull request merged**